### PR TITLE
Bug 2012065: Add summary to etcd alert rules

### PR DESCRIPTION
**Pull request by Elbehery**
This PR adds summary to etcd alert rules. 

According to [Enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/monitoring/alerting-consistency.md#style-guide), all prometheus alerts should have Summary and Description.

### CODE REVIEW

**Comment by Elbehery** [comment]
@hasbro17 the bug was opened against 4.9, so  I added cherry-pick labels to 4.10 and 4.9, please let me know if they are not needed

File: manifests/0000_90_etcd-operator_03_prometheusrule.yaml [hunk 1]
```diff
@@ -109,6 +109,7 @@ spec:
         severity: warning
     - alert: etcdBackendQuotaLowSpace
       annotations:
+        summary: etcd cluster database size is too high for the defined quota.
```
Comment (tjungblu) [inline manifests/0000_90_etcd-operator_03_prometheusrule.yaml:112]: why not take the first line from the description?
> etcd cluster database size exceeds the defined quota

File: manifests/0000_90_etcd-operator_03_prometheusrule.yaml [hunk 2]
```diff
@@ -120,6 +121,7 @@ spec:
         severity: critical
     - alert: etcdExcessiveDatabaseGrowth
       annotations:
+        summary: etcd cluster received 50 percent increase in database writes.
```
Comment (tjungblu) [inline manifests/0000_90_etcd-operator_03_prometheusrule.yaml:124]: I think the alert triggers when the db size increased by 50%, not the number of writes

Comment (Elbehery) [reply manifests/0000_90_etcd-operator_03_prometheusrule.yaml:124]: 👍🏽

**Comment by Elbehery** [comment]
@tjungblu thanks for your review, updated accordingly

**Comment by tjungblu** [comment]
thanks!
/lgtm

**Comment by tjungblu** [comment]
fix for the builds was merged already, but takes some time to propagate: https://coreos.slack.com/archives/CBN38N3MW/p1652347345843039

/retest-required

**Comment by Elbehery** [comment]
@hasbro17  can I get `approve` here ?

**Pull request merged**